.ci/prepare-settings.sh

```diff
@@ -4,14 +4,6 @@ set -e
 mkdir -p .ci-temp
 cp .ci/release-settings.xml .ci-temp/
 
-checkForVariable() {
```

Comment:
Add `source ./.ci/util.sh`

Reply:
Done changes.

Reply:
Not done, recheck push and share testing of this script

Reply:
Done

Reply:
Done

Reply:
![sdfjnsd](https://user-images.githubusercontent.com/71710042/183326111-c629cb31-6533-43d8-9278-8302726d3a56.png)

Testing: Getting the ERROR of checkForVariable. Working.

Reply:
Now I see `source`